package-lock.json

```diff
@@ -3840,61 +3840,46 @@
             }
         },
         "node_modules/@cypress/request": {
-            "version": "3.0.1",
```

Comment:
Are changes in this file needed?

Reply:
It is required by the cypress update, the pipeline would crash without it